eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -169,7 +169,7 @@ jobs:
       Windows_Preconditions_UnitTests_Samples:
         OSVmImage: 'windows-2019'
         vcpkg.deps: 'curl[winssl] paho-mqtt cmocka'
-        VCPKG_DEFAULT_TRIPLET: 'x64-windows-static'
+        VCPKG_DEFAULT_TRIPLET: 'x64-windows-static-md'
```

Comment:
I haven't looked too closely, but this might be interesting to follow up to understand why:
https://github.com/microsoft/vcpkg/issues/1131#issuecomment-303060932

> It is discouraged to use `static-md` triplet...

I wonder if the issue is with the `static` part or the `-md` part.

Reply:
@barcharcraz - can you please take a look at this as well and share your thoughts.

Reply:
That thread is from 2017. Probably there were issues back then. And also I can see that these triplets were added to the officiale vcpkg repo on February, so maybe there were other issues at a time. But now it is there. https://github.com/microsoft/vcpkg/commit/71a9def3df1cd418a8c00e69413f19507ab25fa8.  Right now, it seems permanent and I don't see why we can't use it.
The reason why we were getting the warning was that _we_ built our SDK with dynamic runtime, but cmocka was built with a reference to static runtime, and we were inking them together, and that is what the warning was telling us. Now we build cmocka with the same runtime as our SDK, so now they are consistent, and there is no warning anymore.